core/templates/domain/skill/SkillObjectFactory.ts

```diff
@@ -43,7 +43,9 @@ import { ValidatorsService } from 'services/validators.service';
 import constants from 'assets/constants';
 
 export class Skill {
-  _id: string;
+  // 'SkillId' and 'supersedingSkillId' will be null when creating
+  // interstitial Skill.
+  _id: string | null;
```

Comment:
Would it actually be possible to refactor this object to only be created for skills with ID?

Reply:
should we create an issue to remove the usage of interstitial items in the code? This may be a big refactor

Reply:
Have created an issue #13637 and also a PR [13638](https://github.com/oppia/oppia/pull/13638) to address this